x-pack/plugin/ml/build.gradle

```diff
@@ -12,40 +14,44 @@ esplugin {
   extendedPlugins = ['x-pack-autoscaling', 'lang-painless']
 }
 
-def localRepo = providers.systemProperty('build.ml_cpp.repo').orNull
+boolean useDraWithBuildId = providers.systemProperty(DraResolvePlugin.USE_DRA_ARTIFACTS_FLAG).getOrNull()
```

Comment:
Should we just add this as an extension property in the `DraResolvePlugin` so we aren't having to duplicate this?

Reply:
done